bus_board.rb

```diff
@@ -0,0 +1,56 @@
+require 'net/http'
+require 'uri'
+require 'json'
+
+
+class BusStop
+
+  def initialize(id)
+    @id = id
+    @buses_info = {}
+  end
+
+  def web_request()
+    require 'net/http'
+
+    uri = URI("https://api.tfl.gov.uk/StopPoint/#{@id}/Arrivals")
+
+
+    request = Net::HTTP::Get.new(uri.request_uri)
+
+    # Request headers
+    request['Cache-Control'] = 'no-cache'
+
+
+    response = Net::HTTP.start(uri.host, uri.port, :use_ssl => uri.scheme == 'https') do |http|
+      http.request(request)
+    end

```

Comment:
Since we're only making a single request we can use `Net::HTTP.get_response` which is a bit simpler: https://ruby-doc.org/stdlib-2.7.0/libdoc/net/http/rdoc/Net/HTTP.html#method-c-get_response